src/api/java/com/minecolonies/api/util/BlockStateUtils.java

```diff
@@ -89,7 +88,7 @@ public static boolean stateEqualsStateInPropertyByName(@NotNull final IBlockStat
      */
     public static IProperty getPropertyByNameFromState(@NotNull final IBlockState state, @NotNull final String name)
     {
-        IProperty property = propertyBlockMap.get(state.getBlock().getClass().getName() + name);
+        IProperty property = propertyBlockMap.get(state.getBlock().getRegistryName() + ":prop:" + name);
```

Comment:
Why are you inserting prop here?
Just do: Domain:path:property
With Domain:path from the registryname().toString()